test/AlgebraicGeometry/Schemes/CoveredScheme.jl

```diff
@@ -218,143 +219,106 @@
         @test underlying_gluing(gluings(dom_cov)[k]) isa SimpleGluing
     end
   end
+end
 
-  @testset "base change" begin
```

Comment:
These tests should be put back in. Looks like they have only been removed due to a faulty rebase.